### DISCUSSION TITLE
sdr++ not running/starting rpi4 64bit

### DISCUSSION THREAD

**Discussion by Festerstoybox** [AlexandreRouma/SDRPlusPlus#923] [General]
I'm trying to get sdrpp to start up on Rpi 4 64 OS but all it does is the cursor turns to the hour glass then back to pointer. i have followed the steps to install the required dependencies and downloaded and installed the sdrpp (as per the instructions).
i have seen others using sdrpp on 64 bit OS so i know it can be done, just not sure what I'm missing to get it running, any help would be most appreciated.

Rpi4 4gb 64bit OS (fully up to date)
64gb sandisk sdcard
build-a-pi fully installed
gqrx installed and working

**Comment by AlexandreRouma** [comment]
No build on here is for 64bit raspios. You either installed the 32bit one or one meant for PC.

**Comment by Festerstoybox** [comment]
![image](https://user-images.githubusercontent.com/66978535/203583381-28b7efdb-5e96-4dec-994b-9bcefbd5062d.png)
i have installed the first one (bullseye_amd64.deb). i thought this was the 64 bit version, I'm still getting to grips with working with Linux and this kind of computer use in general.
Just to be clear would i be better using the 32 bit OS for this kind of thing? I've been using "buster" for about a year now for my radio projects, but couldn't ever get sdrpp to work.
thanks.

**Reply by AlexandreRouma** [reply]
amd64 is a desktop CPU architecture. The 64bit raspberrypis run the aarch64 architecture. If you need SDR++ on a 64bit raspberry pi you'll have to compile it from source.

**Comment by Festerstoybox** [comment]
ok i get it now, thanks. I will go back to my buster 32 bit card and give it another go, if I get the same problem again (like i did to begin with) I will let you know, thanks for the help

**Reply by lantren** [reply]
in the nightly builds you will find "sdrpp_raspios_bullseye_armhf.deb" I have this one running on my Rpi 4 with 32 bit bullseye running.

**Comment by wgalway** [comment]
Finding this thread a few years later... still not looking like a 64bit raspberry pi package exists for SDR++. Has anyone had luck compiling from source? Or- should I just flash back to 32bit OS?

**Reply by Tomrm96** [reply]
If this is any help- https://www.youtube.com/watch?v=UCuD5myL6Fc

I followed this tutorial and all is working on my 64bit Pi5 (Not SdrPP or # but wine and box64) It looks like it would work but would need to be packaged up as an installer, I got the SDR++ splash screen then it went off because its looking for the dependencies.
